src/components/Dashboard/Dashboard.jsx

```diff
@@ -51,24 +49,31 @@ export class Dashboard extends React.Component {
     }
     render() {
```

Comment:
This render function is starting to get a bit messy, could we have the different texts/buttons in their own components that can be rendered based on the game status that is in the store?

Reply:
Yeah sure, I'll do this in this pull request